src/languages/spark/spark.formatter.ts

```diff
@@ -144,7 +144,10 @@ export default class SparkFormatter extends Formatter {
       reservedFunctionNames: functions,
       openParens: ['(', '['],
       closeParens: [')', ']'],
-      stringTypes: [{ quote: "''", prefixes: ['X'] }],
+      stringTypes: [
+        { quote: "''", prefixes: ['r', 'X'] },
+        { quote: '""', prefixes: ['r', 'X'] },
```

Comment:
Should use uppercase `R` prefix instead of `r` for consistency with all other prefixes configs.